### ISSUE TITLE
I wrote a few gradle tasks to generate `name.framework`, without pods, >3 votes -> pull request

### ISSUE THREAD

**Issue by Buggaboo** [j2objc-contrib/j2objc-gradle#616]
Hi,

I  read up on how to create ios framework, so I wrote a few gradle tasks using the static libraries and headers generated by j2objc and this project, that automates this process. All without cocoapods.

With >3 votes (i.e. if there's any demand for this) I'm doing a pull request for including these by default in j2objc-gradle.

* [The official documentation](https://developer.apple.com/library/ios/documentation/Miscellaneous/Conceptual/iPhoneOSTechOverview/iPhoneOSFrameworks/iPhoneOSFrameworks.html)
* [Tutorial 1](http://www.insert.io/frameworkios8xcode6/)
* [Tutorial 2](https://robots.thoughtbot.com/creating-your-first-ios-framework)
* [Tutorial 3](http://locomoviles.com/uncategorized/create-ios-cocoa-touch-framework-using-xcode/)
* [Tutorial 4](https://www.raywenderlich.com/65964/create-a-framework-for-ios)

**Comment by erasmospunk** [comment]
This should be useful.

**Comment by Buggaboo** [comment]
Cool, I'll release some code soon.

I'm a bit concerned that this project hasn't been maintained for a while.

**Comment by lekz112** [comment]
@Buggaboo could you please share your tasks? Or point where they are, I wasn't able to find them in your fork.

**Comment by Buggaboo** [comment]
@lekz112, I totally forgot about this. I'll cook up something soon.

**Comment by Buggaboo** [comment]
I'm creating a new smaller j2objc-gradle-plugin project from scratch, stay tuned...

**Comment by brunobowden** [comment]
@Buggaboo - would you be interested in taking over the existing repository? It would make sense to have a consolidated place for activity.

**Comment by Buggaboo** [comment]
@brunobowden yes, I guess I could spare some time now and then to do minor fixes etc. I am interested in taking over.
